app/views/usages/_usage.html.erb

```diff
@@ -0,0 +1,8 @@
+<tr>
+  <td><%= usage.subscription.user.name %></td>
+  <td><%= link_to usage.subscription.plan.name, usage.subscription.plan %></td>
+  <td><%= usage.feature.name %></td>
+  <td><%= usage.subscription.plan.features.include?(usage.feature) ? usage.feature.max_unit_limit : '' %></td>
```

Comment:
what is the purpose of this line?

Reply:
This is just to be on safe side and avoid null exception, checking if usage subscription plan has that feature which limit we are going to view.